Cargo.toml

```
@@ -0,0 +1,34 @@
+[package]
+name = "tree-sitter-redscript"
+description = "Redscript grammar for tree-sitter"
+version = "0.1.0"
+authors = ["Jekky"]
```

```suggestion
authors = ["jekky"]
```
and you can list yourself in `authors`